bin/terminus

```diff
@@ -21,20 +21,19 @@ if ($phar_path) {
 }
 
 use Pantheon\Terminus\Config;
-use Pantheon\Terminus\Runner;
-use Robo\Robo;
-use Symfony\Component\Console\Application;
+use Pantheon\Terminus\Terminus;
 use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Console\Output\ConsoleOutput;
 
-// Configuring the dependency-injection container
+// Create the input and output objects for Terminus to run against.
 $input = new ArgvInput($_SERVER['argv']);
 $output = new ConsoleOutput();
+// Create a config object.
 $config = new Config();
-$application = new Application('Terminus', $config->get('version'));
-$container = Robo::createDefaultContainer($input, $output, $application, $config);
 
 // Running Terminus
-$runner = new Runner($container);
-$status_code = $runner->run($input, $output);
+// @TODO: Figure out if and why we need to pass input/output twice and define what happens if we
```

Comment:
Input and output are provided in the beginning because some objects in the container need a reference to input and output. This is undesirable; with enough work, this requirement could be removed. But Robo was all procedural, and there are a couple of places where globals are used for I/O to maintain backwards compatibility. It might be fair to not pass in input and output when creating the container if you were careful to not rely on any of the backwards-compatible code. However, that wouldn't necessarily be good or safe; errors resulting from this might be hard to diagnose. Ultimately, we simply do not want to have any variable references to input or output originating from the container, or container construction.

Input and Output are passed in to `run` rather than being fetched from the container because this is how we'd ultimately like it to work.